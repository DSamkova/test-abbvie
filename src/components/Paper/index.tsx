import {Paper as MuiPaper, PaperProps, SxProps} from "@mui/material";
import {styled} from "@mui/material/styles";

const Paper = styled(MuiPaper)<PaperProps & SxProps>(() => ({
    backgroundColor: "rgb(255, 255, 255)",
    color: "rgb(33, 43, 54)",
    transition: "box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms",
    backgroundImage: "none",
    overflow: "hidden",
    position: "relative",
    boxShadow: "rgba(145, 158, 171, 0.2) 0px 0px 2px 0px, rgba(145, 158, 171, 0.12) 0px 12px 24px -4px",
    borderRadius: "16px",
    zIndex: "0",
    display: "flex",
    alignItems: "center",
    padding: "24px",
}));

export default Paper;