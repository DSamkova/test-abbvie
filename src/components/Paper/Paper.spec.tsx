import React from 'react';
import { render } from '@testing-library/react';
import Paper from './Paper';

describe('Paper Component', () => {
    test('matches snapshot', () => {
        const { asFragment } = render(<Paper />);
        expect(asFragment()).toMatchSnapshot();
    });
});
