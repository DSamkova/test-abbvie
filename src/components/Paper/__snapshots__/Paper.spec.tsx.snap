// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-g7f0bg-MuiPaper-root"
  />
</DocumentFragment>
`;
